### DISCUSSION TITLE
How to extract lines in PDF?

### DISCUSSION THREAD

**Discussion by yooongchun** [pymupdf/PyMuPDF#996] [Q&A] [answered]
How can I extract lines by PyMuPDF?

Thanks a lot!

**Comment by JorjMcKie** [comment]
This is a typical ``Discussions`` item, so I am going to transfer it to there, category "Looking for Help".
I will add a few hints after his.

**Comment by JorjMcKie** [comment, marked as answer]
Method `page.get_drawings()` returns a list of "paths", which are dictionaries each containing any number of elementary drawing commands like curves, rectangles and lines.
Please consult the documentation!
